mdsuite/database/data_manager.py

```diff
@@ -317,10 +334,10 @@ def dictionary_generator(ensemble_loop, correlation_time, data_range):
                     if item == str.encode("data_size"):
                         pass
                     else:
-                        if system:
-                            output_dict[item] = glob_data[item][start:stop]
-                        else:
-                            output_dict[item] = glob_data[item][:, start:stop]
+                        # if system:
```

Comment:
It seems this is quite a critical bug. If there were more than 1 ensembles over the data with a system-wide property this failed. This means that currently the main branch system calculators are broken.

Reply:
If this is resolved, can the old code be removed?

Reply:
yes will do that

Reply:
94fd6e1